docs/get-started/getting-started-databricks.md

```diff
@@ -155,3 +155,32 @@ storage location using Databricks [cluster log
 delivery](https://docs.databricks.com/clusters/configure.html#cluster-log-delivery-1).  Enable this
 option before starting the cluster to capture the logs.
 
+## Limitations
+
+1. Adaptive query execution(AQE) and Delta optimization write do not work. 
```

Comment:
I think it's important to emphasize here that queries may still see significant speedups even with AQE disabled.  Some users may see this and think, "well I run with AQE on, so this is a non-starter" which is not necessarily the case.

Reply:
@jlowe Just added one line for this. How about now?